### PR TITLE
Add page separator, fix responsiveness

### PR DESCRIPTION
**Pull request by ninaandal**
For the responsiveness, I took some artistic freedoms as to how I thought it was best, but feel free to change it if you disagree

### CODE REVIEW

File: components/pages/page/Page.tsx [hunk 1]
```diff
@@ -20,14 +20,17 @@ export function Page({ data }: { data: PagePayload }) {
...
-      {body && <PortableText components={components} value={body} />}
+        {/* Body */}
+        {body && <PortableText components={components} value={body} />}
 
-      {/* Workaround: scroll to top on route change */}
-      <ScrollUp />
+        {/* Workaround: scroll to top on route change */}
+        <ScrollUp />
+      </div>
+      <div className="absolute left-0 w-screen border-t" />
```
Comment (RitaDias) [inline components/pages/page/Page.tsx:33]: Looks to me that this should probably be in the footer :)

Comment (ninaandal) [reply components/pages/page/Page.tsx:33]: But it's not supposed to be on the front page

Comment (RitaDias) [reply components/pages/page/Page.tsx:33]: I know that it doesn't have it in the design (on the home page) but it looks like it exists on both pages and on project details pages 🤔 hence why I felt it made sense to either have them on all or none, feels kinda strange to duplicate this line

Comment (ninaandal) [reply components/pages/page/Page.tsx:33]: I had it before on all pages, but then Rob removed it on all pages - whatever you guys want to do, I am fine with. But this is correct according to figma

Comment (RitaDias) [reply components/pages/page/Page.tsx:33]: @robinpyon booooo 
Alright, the line stays 😔

**Review by robinpyon — APPROVED** [review]

**Pull request merged**